src/docs/tutorials/proximity-steering.md

```diff
@@ -0,0 +1,203 @@
+
+- Have **SSL/TLS** mode set to **Full**.
+
+    **SSL/TLS → Overview → Full**
+
+- Have **Always Use HTTPS** enabled.
+    
+    **SSL/TLS → Edge Certificates → Always Use HTTPS**
+    
+    This ensures that Railway avoids managing the insecure redirect, which would otherwise lead to an incorrect redirection to an upstream endpoint.
+
+- Have a `/health` or similar endpoint that returns a status code of 200.
+
+    This allows Cloudflare to check the health of our Railway services so they can handle region failover. As a bonus this can also be used on Railway to achieve [zero-downtime deployments](/reference/healthchecks).
+
+
+## 1. Creating a Health Monitor
+
+- Open the Load Balancing page.
+
+    **Traffic → Load Balancing**
+
+<Image src="https://res.cloudinary.com/railway/image/upload/v1722015860/docs/tutorials/proximity-load-balancing/load_balancing_page_yn5bm8.png"
+alt="screenshot of the load balancing page"
+layout="responsive"
+width={1060} height={555} quality={100} />
+
+- Click **Manage Monitors** and then **Create**.
+
+- Enter your desired name for this health monitor.
+
+- Choose **HTTPS** as the type.
+
+- Enter your health endpoint path
+
+    Example - `/health`
+
+- Leave Port 443 as the default.
+
+<Image src="https://res.cloudinary.com/railway/image/upload/v1722015787/docs/tutorials/proximity-load-balancing/health_monitor_oty6pd.png"
+alt="screenshot of the cloudflare health monitor"
+layout="responsive"
+width={1060} height={315} quality={100} />
+
+- Click **Save**.
+
+## 2. Creating the Pools
+
+- Go back to the Load Balancing page.
+
+- Click **Manage Pools** and then **Create**.
+
+- Fill out the name and description and leave **Endpoint Steering** as its default, it will not be used with only a single endpoint.
+
+- Enter the endpoint name, using the service name is ideal.
+
+- For the **Endpoint Address** we use the Railway generated domain.
+
+    Example -  `region-us-west1.up.railway.app`
+
+    This should only be the domain, excluding both the scheme and trailing slash.
+
+- For the weight option we will use a value of **1**.
+
+- Click **Add host header** and enter the same value as used for the Endpoint Address.
+
+    Railway does host based routing so we need to set the host header so they know how to route the incoming requests from Cloudflare.
+
+- Remove the second empty endpoint.
+
+    Our pool only needs to contain a single endpoint as Railway handles single region replicas for us.
+
+<Image src="https://res.cloudinary.com/railway/image/upload/v1722015878/docs/tutorials/proximity-load-balancing/pool_settings_config_qh5s1k.png"
+alt="screenshot of end endpoint settings in the pool creator"
+layout="responsive"
+width={1060} height={600} quality={100} />
+
+- Click **Configure coordinates for Proximity Steering** and enter the Latitude and Longitude for your service region that can be found in this [JSON file](https://www.google.com/about/datacenters/json/locations.json).
```

Comment:
how about adding an example of how to do this?  i opened that json file, and assumed i was looking for like us-west but don't see that, so confused

Reply:
that's fair, I will add that